packages/store/src/module.ts

```diff
@@ -89,6 +89,15 @@ export class NgxsFeatureModule {
   }
 }
 
+type ModuleOptions = Partial<NgxsConfig>;
+
+export function ngxsConfigFactory(options: ModuleOptions): NgxsConfig {
+  const config = Object.assign(new NgxsConfig(), options);
```

Comment:
Why not just return directly rather than assign a variable and return that in the next line?